manifests/external-dns-operator_clusterserviceversion.yaml

```diff
@@ -0,0 +1,121 @@
u48FTmvNSl3dmQ+OJIWszkuO+duXFpcTkzO3Zy4mANbP41vrBAHb+tQ9Wm6gm4cHN2XDTjdXsHzYXNezco41tEQ+sLpbkxe/YE0CPlNXEx2Ttw3OZu916+Lb00/Fl2+gw7olSIvERu8WcEGwfFEXsG9Ur862FBE12B6WjN/dXXKS8vOzjuwMDs7e/OFAxcubM5OW+l9u8E/H76/1h2we8OlfE5EirsxwWT7cCIEXrCYFAJIy+iG0CPeTtn139XHrCyMq/+9Ps21MO73378pdNr6LD4ANtBtHjD4Ii+v4DV8tnG546+p8/OZBepWw19Lz/zUybUm5sOcuI/vLiypv/uvVXlpIC4mptDJ+5V/AWyl5d0QLsmP72c0VXzki7ph2+1tg+141+nb3hBu1ZaV3/1rS0lOzYWP09I+zCmsgfV35BAaTWEyD/AivLzW0E0oBAWkZsQ1eKlphLFE4Q5419cfgHB5H6etjFv1XTas9603VlQsYEWtjQVsaSzV5LoTPS0vF/qvW++Keeddj3kH4WpcN39XcqDG1enT+G3CCuFaP/OUuai5U5bhv24rBrc5uyRnodq7vC1xH+ZhcDvDnoTtlJoLTtbZoYK7u6qkpH7LwpIYFO5CSUnJNyudVHCCVnPBeT4tqojGgulaeGHzhUIn1//968VfBwpX5m3Og2goXIewInqtuzng6L33r7EWRDV27d3q5O1UU1Ozfu/D7w8fvvrtq/XwCSRmzZEHmRULFkRVc0yFk0QkXy37+us9LGHFyZdXDz7cu97Je+/BsvxTFae2RT8/eNsJYt1+ePD5yV1VIcKe72UCTw7PeDi2LP76zVtff/Vo57EdtXfcciN3fP/w1TNlclF19cuT0WHFB1+9eli6+kqu250/Mo7t/vrbW7/d3J6iMBpOkvxT4q57kW17Iu+4YS/84w58In7enl9eVFRVcHjPnrZUtznqd2r3tF2517H6idRoOKpoUcGxf2DkFKiue5exG8AnyWFNTQnhaO/n+Pb7xurTTRLjc8erky/FkiG7XDzXw89D3tJdpxCJJMpOpUDAVwQH+4Qe5hcL+FJBsRruMN8kZkpVcOKWsvBQ5aHgltJHXl1d3T33PVvKSru7urxCL0d0Hbp6uUuuhlMCs8CdKJOGikKDe5ShV1uuBvf0cno8y0rh0aHwiFul4uKeUhUjm80DlyK7H3w4tEzZFbqo99Cj8PucylLlZY9DjzidoRGPyjyD++D2mAyHUYXn5pt6pdaH6gPXxbF+vNx7R1MVChDL5/Hgk9rI1Dp1ue3xMA2Ovz0V1oDbvay1uwt2B7TdY6NP2rb33Ojp+OMO/Ibv0ebW3QWN6X/cgafle2VnsWlwCkH19ub2gO0JXk0hwordf+5qb68+/edvDx603Dx9OqA5o+B0a5VwW6nn0u3pzRnH1vZtTRjdMyWdKQldclhM8qUhCzpSyloDqhv8/dElSUNBQMBL8eF8521NcGVXd79bdt/d5BYt4VeKOVJ0pbhmbXm61/YQdMFVlNFRlS90Zi2okh2r2pkAT0YR2yuWcvxMhqN7iKXFMuy065JbT0Wrl5Od7arlZFFrdQRKE58ETqUs3OQLkCJ4UaxcoM7Iop+/DciH60mWMFkUAB+chT+U3Xiies9PrHT38nQzCG4mk/h44C0jvgebLlWznH75q//5tmntNhjHUzuLoou2tVb/uPcX2etdAp6bJ8eA1atqba5/B0F560RLBM+vOEUWFhbWlLzDM0ny759P/Njbbxk+4Np8v4vG2pyEm86wClQAPZu0Pi/jMw+pHGXzFDws9rzQm9efx+rbtkV3xYZpbHTYfmB95hN9+yoR8cevJ7z54uH447vkevdVFhOZ9hpwI+0Y5FqgZ3ddEPBUfTvC5rF5PNVCINz/h9oUPXvEQBFkxZj+96YYCfc+BTmrb5MqqXORSIWRVMwJ71UtV3kJ1eK5evbcQeQ8xkSN/W9LnD1MHg/oGYvEStVLm17PYqVcXWS8JL7e1G1CKDP67WfajmMSzwFDxwe9nh5yebihEwwwH8bSXnO7FkaTSVwMDN3Sr+z1kM+tNHScAL5EKBOHa+4vknBTaUxilqHuUalKOZVqqHO50Lk3tvYtcDMYVkESg/1TyoHBziUilGm2/QclFrhR4yhcGE4D8ZI4BqKxwUUugTb5zbkFCTeZRnC5pGMY/9ZY1wCbwwZZZDxjwttDIAvcaBoBSfwQmGG6pSHAyJpnzRhvOxDcyBHQv8USdDJlFkPHcp9x8Yxxk3ADjEQtcEMn0AjcoCwTJ5OaY/vFCJ4xbTpuwAEs9A/G04q72JS5q8ZsS3KWRiTQ7EbhSFqFWQ6qqXKuoVzQziTHjUEueApt9FAcSccY21Y9M6caylFtgpLZmKBkDE6nMk8th7FCHtcOWhEwgDJgpL5BfZ/Yh6hN7GPI0PyjLxA8k2aYGAiVMo1GpUyUz3wHPdBGU35ucFIntVCLgHcJzBqc7AEl/1FMyDXRnmS4ZkUlQ0NT+PnRQcy1IbnmbzJKpvZaZDfvUq6BKYRxZG9cbqzeCBXkqFRUF3kGpBCNQe0SxBojvzFqKkwjNhbTiJ3XqhHT1FRsIJPVWjHjpFsw/JYOqAKOqKcoUNHP/jMIgYkfPdwUJVyfvo9L2a9dLQNPhK7W+U0yUefXp17EuwS10Qfsa2iYryxGUBXjEDOoGNHYqItiIKUTfCH3EtqxUPKTzKHRRItCreNCpSyO/Xsx7yJG/hm25hOEqnW1eOQL2NfmzKJSqY6OVCqqNHE8F4gMWodmUAqnYkURdDb173jmblyCquymOZhdPQwjRXKYyKBYI7TZmy6ei4w8v3H/vi8QMpOBtxn+LrTRmH5yGiqLRhAiNK4L0ZrCwJtbNtwvosMdRnzAoFCYDFT5zWTYDXtXYH1FgbO1t5kw1m683YjRkyeNxJmH/PoE+33Jsnzn/xGwsFRhwEfSu/9DQt894v8Xkkn2f1t20PFmJc7vAAAAAElFTkSuQmCC'
```

Comment:
This is https://github.com/kubernetes-sigs/external-dns/blob/master/img/external-dns.png for those following along

Reply:
Yeah, I tried with this one first but converting it to base64 gave me a string so long that my vim broke up :)
I'll think of how to get a smaller version of this PNG.